docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -14,31 +14,72 @@ that you can easily store Hive tables in Alluxio's tiered storage.
 
 ## Prerequisites
 
-The prerequisite for this part is that you have
-[Java](Java-Setup.html). Alluxio cluster should also be
-set up in accordance to these guides for either [Local Mode](Running-Alluxio-Locally.html) or
-[Cluster Mode](Running-Alluxio-on-a-Cluster.html).
-
-Please [Download Hive](http://hive.apache.org/downloads.html).
+* [Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.
+* [Download and setup Hive](https://cwiki.apache.org/confluence/display/Hive/GettingStarted).
+* An Alluxio cluster has been set up and is running according to either
+[Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+* Make sure that the Alluxio client jar is available.
+  This Alluxio client jar file can be found at `{{site.ALLUXIO_CLIENT_JAR_PATH}}` in the tarball
+  downloaded from Alluxio [download page](http://www.alluxio.org/download).
+  Alternatively, advanced users can compile this client jar from the source code
+  by following the [instructions](Building-Alluxio-From-Source.html).
 
 To run Hive on Hadoop MapReduce, please also follow the instructions in
 [running MapReduce on Alluxio](Running-Hadoop-MapReduce-on-Alluxio.html) to make sure Hadoop
 MapReduce can run with Alluxio.
 
 ## Configure Hive
 
-We recommend you to download the tarball from
-Alluxio [download page](http://www.alluxio.org/download).
-Alternatively, advanced users can choose to compile this client jar from the source code
-by following Follow the instructs [here](Building-Alluxio-From-Source.html#compute-framework-support).
-The Alluxio client jar can be found at `{{site.ALLUXIO_CLIENT_JAR_PATH}}`.
+### Basic Setup
 
+Include Alluxio client jar to ensure that Hive can query and access data stored on Alluxio.
 Set `HIVE_AUX_JARS_PATH` either in shell or `conf/hive-env.sh`:
```

Comment:
I'd remove "in shell". Giving basic users more options add more mental overhead and can cause confusion. Advanced users will already know that they can set things in-shell instead of in the `hive-env`.

Reply:
done